ironfish-cli/src/commands/accounts/notes.ts

```diff
@@ -28,39 +28,46 @@ export class NotesCommand extends IronfishCommand {
 
     const client = await this.sdk.connectRpc()
 
-    const response = await client.getAccountNotes({ account })
+    const response = client.getAccountNotesStream({ account })
+    let first_response = true
```

Comment:
We usually use camelCase throughout our codebase. Can you update this variable and `no_header` below?